Blazor.Test/Pages/WetComponent.razor

```diff
@@ -0,0 +1,31 @@
+﻿@inject IJSRuntime jsRuntime
+@page "/wetform"
+
+<PageTitle>Wet Component</PageTitle>
+<h3>Wet Component</h3>
+
+<WetContainer WetComponentName = "@wetComponentToRefresh">
```

Comment:
When trying the page, if you leave fields empty and click "Submit" the errors appear OK but when you click on the error it brings you to the index page instead of anchoring to the field on the current page.    Tried the same in React version and it works, but I'm not sure what is different (I don't have an "action" in React sample... could that be it?)

Reply:
I figured out the problem but trying to find a solution...

Reply:
Forgot to msg earlier but this was fixed